catalog/app/utils/usePrevious.js

```diff
@@ -1,10 +1,11 @@
 import * as React from 'react'
 
-export default (value, onChange) => {
+export default function usePrevious(value, onChange) {
```

Comment:
why do you think "callback" is better in this context?

Reply:
`onChange` is expected name for `<input />`
`usePrevious` returns previous value asynchronously - it's a natural pattern for nodejs environment. And usually `callback` used for that, so this name is expected (not quite, because callbacks returns error too)

Is your intention was to call function after _changing_ ref, and that's why you choose onChange?

Reply:
it's not used to return previous value asynchronously -- actually, the value is _returned_ synchronously, but the onChange callback is meant to be called when the value (possibly) changes, so that the caller can compare previous and current values and run some (possibly effectful) code, so it's more like `onPossibleValueChange`